docker/docker.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+trap "echo; exit" INT
+trap "echo; exit" HUP
+
+PARENT_DIR=$( echo $(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")") )
```

Comment:
Hey Luke, nice PR thank you for your effort.

`realpath` by default is not installed on MacOS, I had to run `brew install coreutils`, this might be helpful for other people.

An alternative approach could be a check to see if the OS is macOS (Darwin as returned by uname), it will use the Python code to resolve the path.

```bash
function realpath() {
    if [[ "$(uname)" == "Darwin" ]]; then
        python -c "import os; print(os.path.realpath('$1'))"
    else
        echo "$1"
    fi
}
```

Reply:
thanks, that's very interesting feedback and an interesting solution. i initially used that approach here https://github.com/paritytech/substrate/commit/7c3d27927577b7fa9ea659aed3197d1f67dd663e since their previous approach wasn't handling symlinks, and i've since used that approach elsewhere.

it seems `os.path.realpath` also handles symlinks.

i also think most operating systems would already have python installed, and your script uses very basic python syntax so it would be fine with those with python 2.7 and recent versions, but for example i'm on a new macOS machine and i didn't have the `python` binary, but i had `python3` instead.

another possible option is to do it like what is done here https://getsubstrate.io/, and installing as shown here https://docs.brew.sh/Installation, 
```
if [[ "$OSTYPE" == "darwin"* ]]; then
	set -e
	echo "Mac OS (Darwin) detected."

	if ! which brew >/dev/null 2>&1; then
                 echo "Installing Homebrew..."
		/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
	fi

	brew update
	brew install coreutils
fi
```
but they might be prompted to enter a password if they're not an admin
```
Checking for `sudo` access (which may request your password)...
Password:
```

Reply:
Yeah, installing `coreutils` is also a possibility, it is my preference to maybe ask before installing it because `coreutils` is installing multiple binaries but, up to you.

Reply:
i've pushed this commit to prompt macOS users to press SPACE if they want to proceed with the script and install the necessary dependencies https://github.com/a16z/helios/pull/262/commits/445f2cee2ef35ded3ae26803ea02af63e21a946f